samples/python/kalman.py

```diff
@@ -86,7 +86,7 @@ def draw_cross(center, color, d):
 
             cv2.imshow("Kalman", img)
 
-            code = cv2.waitKey(100)
+            code = cv2.waitKey(100) & 0xFF
```

Comment:
Just covered this with @berak and it seems that due to the inline nature, there will be nothing left after the compiler to wrap. His suggestion is to seperate declaration and implementation, to be able to provide the same functionality in Python? @terfendail what you think?